app/Database/Schema/AccountSchema.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Database\Schema;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @method null|string getAccountId()
+ * @method null|string getAccountNumber()
+ * @method null|string getSubscriptionType()
+ * @method self setAccountNumber(string $accountNumber)
+ * @method self setSubscriptionType(string $subscriptionType)
+ */
+trait AccountSchema
+{
+    /**
+     * @ORM\Column(type="string", name="account_number")
+     *
+     * @var string
+     */
+    protected $accountNumber;
+
+    /**
+     * @ORM\Column(type="string", name="subscription_type", columnDefinition="ENUM('in:monthly', 'lifetime')")
```

Comment:
columnDefinition is great, but on our end we don't use it since we are already validating this on the application level. The reason is if this changes we don't want to modify the database as much as possible.